core/src/test/java/org/apache/gravitino/cache/TestCacheConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.cache;
+
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestCacheConfig {
+  @Test
+  void testDefaultCacheConfig() {
+    CacheConfig cacheConfig = new CacheConfig();
+
+    Assertions.assertEquals(10_000, cacheConfig.getMaxSize());
+    Assertions.assertTrue(cacheConfig.isExpirationEnabled());
+    Assertions.assertEquals(60, cacheConfig.getExpirationTime());
+    Assertions.assertEquals(TimeUnit.MINUTES, cacheConfig.getExpirationTimeUnit());
+    Assertions.assertFalse(cacheConfig.isCacheStatusEnabled());
+    Assertions.assertTrue(cacheConfig.isWeigherEnabled());
+    Assertions.assertEquals(200_302_000, cacheConfig.getMaxWeight());
+  }
+}
```

Comment:
You don't have tests for caffeine cache, only add the trivial tests for configuration, the test coverage is not enough.